Makefile.inc

```
@@ -703,7 +703,7 @@ install-git-commit-clangfmt:
 include util/crossgcc/Makefile.inc
 
 .PHONY: tools
-tools: $(objutil)/kconfig/conf $(objutil)/kconfig/toada $(CBFSTOOL) $(objutil)/cbfstool/cbfs-compression-tool $(FMAPTOOL) $(RMODTOOL) $(IFWITOOL) $(objutil)/nvramtool/nvramtool $(objutil)/sconfig/sconfig $(IFDTOOL) $(CBOOTIMAGE) $(AMDFWTOOL) $(AMDCOMPRESS) $(FUTILITY) $(BINCFG) $(IFITTOOL) $(objutil)/supermicro/smcbiosinfo $(ECCTOOL) $(SBSIGNTOOLS)
+tools: $(objutil)/kconfig/conf $(objutil)/kconfig/toada $(CBFSTOOL) $(objutil)/cbfstool/cbfs-compression-tool $(FMAPTOOL) $(RMODTOOL) $(IFWITOOL) $(objutil)/nvramtool/nvramtool $(objutil)/sconfig/sconfig $(IFDTOOL) $(CBOOTIMAGE) $(AMDFWTOOL) $(AMDCOMPRESS) $(FUTILITY) $(BINCFG) $(IFITTOOL) $(objutil)/supermicro/smcbiosinfo $(ECCTOOL) $(SBSIGNTOOLS)/create-container
```

```suggestion
tools: $(objutil)/kconfig/conf $(objutil)/kconfig/toada $(CBFSTOOL) \
	$(objutil)/cbfstool/cbfs-compression-tool $(FMAPTOOL) \
	$(RMODTOOL) $(IFWITOOL) $(objutil)/nvramtool/nvramtool \
	$(objutil)/sconfig/sconfig $(IFDTOOL) $(CBOOTIMAGE) \
	$(AMDFWTOOL) $(AMDCOMPRESS) $(FUTILITY) $(BINCFG) $(IFITTOOL) \
	$(objutil)/supermicro/smcbiosinfo $(ECCTOOL) $(SBSIGNTOOLS)/create-container
```
This line is very long, what makes it harder to read. Maybe try to split it

---

I don't think all `make`s behave properly when it comes to split lines. Upstream keeps it in one line so probably better to leave is as it were.